apps/src/music/views/MusicPlayView.tsx

```diff
@@ -0,0 +1,109 @@
+import React, {memo, useCallback, useMemo} from 'react';
+
+import {Button} from '@cdo/apps/componentLibrary/button';
+import {Heading2, BodyTwoText} from '@cdo/apps/componentLibrary/typography';
+import Link from '@cdo/apps/componentLibrary/link';
+import commonI18n from '@cdo/locale';
+import codeOrgLogo from '@cdo/static/code-dot-org-white-logo.svg';
+
+import moduleStyles from './music-play-view.module.scss';
+
+import musicI18n from '../locale';
+
+interface MusicPlayViewProps {
+  isPlaying: boolean;
+  onPlay: () => void;
+  onStop: () => void;
+  projectName: string;
+}
+
+const MusicPlayView: React.FunctionComponent<MusicPlayViewProps> = ({
+  isPlaying,
+  onPlay,
+  onStop,
+  projectName,
+}) => {
+  const shareData = useMemo(
+    () => ({
+      title: projectName,
+      url: window.location.href,
+    }),
+    [projectName]
+  );
+
+  // Share button will only appear when users browser supports the Web Share API
+  // (Can be mobile browsers and some desktop browser like macOS Safari)
+  // Requires HTTPS connection (adhoc or production page).
+  // Was unable to access it on localhost with our routing (http://localhost-studio.code.org:3000/
+  // is not considered a sage URL by browser, only http://localhost:3000/ is).
+  const canShare =
+    navigator && navigator.canShare && navigator.canShare(shareData);
+
+  const onShareProject = useCallback(() => {
+    navigator?.share(shareData);
+  }, [shareData]);
+  const onViewCode = useCallback(() => {
+    console.log('view code');
+  }, []);
+  const onRemix = useCallback(() => {
+    console.log('make my own');
+  }, []);
+
+  return (
+    <div className={moduleStyles.musicPlayViewContainer}>
```

Comment:
This is probably more a question for @markabarrett - do we want the Project title and rename/share/remix buttons to be shown in the header? 

<img width="931" alt="Screenshot 2024-04-24 at 3 04 39 PM" src="https://github.com/code-dot-org/code-dot-org/assets/107423305/0ee39bf4-ffbf-401d-8579-4d86c557e4e3">

Or should the overlay cover the header as it does in other lab share views?
<img width="1426" alt="Screenshot 2024-04-24 at 3 05 22 PM" src="https://github.com/code-dot-org/code-dot-org/assets/107423305/ebfea856-090c-4a76-9b82-0d00ded272a3">

Reply:
In the actual share view, the header will be hidden (like it is in the dance project you screenshotted for example)

Reply:
Do we want the Code.org logo showing as well on the left upperhand corner?

Reply:
The `Overlay` component is used within the `StudioAppWrapper`. Wondering if we would want to use this `Overlay` or not.

Reply:
Nice! I see this hidden header and Code.org logo in your PR - https://github.com/code-dot-org/code-dot-org/pull/58214

Reply:
> The `Overlay` component is used within the `StudioAppWrapper`. Wondering if we would want to use this `Overlay` or not.

Note that we want to take the opportunity to build everything in a modern fashion with Lab2, and do not want to use `StudioAppWrapper` or related components if at all possible.

Reply:
>In the actual share view, the header will be hidden (like it is in the dance project you screenshotted for example)

FYI - on a different branch, I merged Denys's branch with Sanchit's [WIP branch](https://github.com/code-dot-org/code-dot-org/tree/sanchit/lab2-enable-share-view) - 
![Screenshot 2024-04-25 at 10 47 41 AM](https://github.com/code-dot-org/code-dot-org/assets/107423305/c3bd91e3-8638-4f4b-9238-30b79c1d39d0)